src/Components/ServiceScene/Breakdowns/LabelValuesBreakdownScene.tsx

```diff
@@ -101,6 +120,51 @@ export class LabelValuesBreakdownScene extends SceneObjectBase<LabelValueBreakdo
         }
       }
     }
+
+    // If we're in an error state
+    if (newState.data?.state === LoadingState.Error && this.activeLayoutContainsNoPanels()) {
+      const activeLayout = this.getActiveLayout();
+      // And the active layout is not showing panels
+      if (activeLayout instanceof ByFrameRepeater) {
+        const errorState = this.getErrorStateAlert(newState.data.errors);
+        // Replace the loading or error state with new error
+        activeLayout.state.body.setState({
+          children: [errorState],
+        });
+      }
+    }
+  }
+
+  private getActiveLayout(): ByFrameRepeater | SceneFlexLayout | undefined {
+    const layoutSwitcher = this.state.body;
```

Comment:
I think you can just check if (!this.state.body) { return undefined; } and simplify a bit.

Reply:
yeah but then I think we need a type assertion